### PR TITLE
#157468881 Redirect user to dashboard if logged in

### PR DESCRIPTION
**Pull request by bahdcoder**
#### What does this PR do?

Redirects user to dashboard if logged in

#### Description of Task to be completed?
     - add redirect component on the authentication page
     - add auth util for checking user authentication status
     - add tests for user redirect on logged in

#### How should this be manually tested?

Visit the authentication screen, and log in with a valid andela google account.
Try visiting the authentication screen again.

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#157468881](https://www.pivotaltracker.com/story/show/157468881)

#### Questions:
None

### CODE REVIEW

**Comment by andela-kakpobome** [comment]
1. Remove the space between `feat` and `(`
2. Since this is a `feature`, you should add the `Finishes #<story-id>` footer, not `Maintains`

File: src/pages/AuthPage/AuthPage.test.tsx [hunk 1]
```diff
@@ -29,7 +32,14 @@ describe('Auth page', () => {
     const wrapper = shallow(<AuthPage { ...props } />);
     const redirectUrl = `${props.ANDELA_AUTH_HOST}/login?redirect_url=${props.AUTH_REDIRECT_URL}`;
 
-    wrapper.find('button').simulate('click');
+    wrapper.find('.login-btn-container').simulate('click');
     expect(window.location.replace).toHaveBeenCalledWith(redirectUrl);
   });
+  it('redirects user to dashboard if user is logged in', () => {
```
Comment (andela-kakpobome) [inline src/pages/AuthPage/AuthPage.test.tsx]: Add line spaces before each test case

File: src/pages/AuthPage/index.tsx [hunk 2]
```diff
@@ -27,25 +31,31 @@ class AuthPage extends React.Component<IAuthProps> {
...
-                <img src="images/google-logo.png" />
-              </button>
-              <button className="login-button">
-                LOGIN WITH GOOGLE
-              </button>
+      <React.Fragment>
+        {
+          authService.isAuthenticated()
+          ? <Redirect to="/dashboard" />
+          : <div className="auth-container">
```
Comment (andela-kakpobome) [inline src/pages/AuthPage/index.tsx]: Indent `37, 38`

File: src/pages/AuthPage/AuthPage.test.tsx (hunk 1 above)
Comment (bahdcoder) [reply src/pages/AuthPage/AuthPage.test.tsx]: Resolved.

File: src/pages/AuthPage/index.tsx (hunk 2 above)
Comment (bahdcoder) [reply src/pages/AuthPage/index.tsx]: Resolved.

**Pull request merged**